source/backend/entities/Partials/AcquisitionFile.cs

```diff
@@ -10,6 +11,11 @@ public partial class PimsAcquisitionFile : StandardIdentityBaseAppEntity<long>,
         #region Properties
         [NotMapped]
         public override long Internal_Id { get => this.AcquisitionFileId; set => this.AcquisitionFileId = value; }
+
+        public bool PersonIsAssignedToFile(long personId)
```

Comment:
hmm, I think I'm a bit uncomfortable with this being on the entity directly, since there is no guarantee that the child entities have been loaded.

I think we need an extension method that:
1) always calls the acquisition repository in a manner that ensures the acqpeople are loaded, probably using the file id.
2) checks if the person id is assigned.

If not we would be relying on the caller to know that they need to pass an acq file with the people loaded, otherwise we would return a false negative.

Reply:
moved the check to the extension method.